src/audio/drc/drc_math_lib/asin/c_fixpt/typedef.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * File: typedef.h
+ *
+ * Author :Shriram Shastry
+ * Copyright (c) 2016, Intel Corporation All rights reserved.
+ */
+
+#ifndef TYPEDEF_H
+#define TYPEDEF_H
+#ifndef __cplusplus
+#include <stdbool.h>
+#endif
+
+
+
+/*=======================================================================*
+ * Fixed width word size data types:                                     *
+ *   int8_T, int16_T, int32_T     - signed 8, 16, or 32 bit integers     *
+ *   uint8_T, uint16_T, uint32_T  - unsigned 8, 16, or 32 bit integers   *
+ *   real32_T, real64_T           - 32 and 64 bit floating point numbers *
+ *=======================================================================*/
+typedef signed char int8_T;
```

Comment:
why not just use inttypes.h?

Reply:
ack, @ShriramShastry please use stdint.h